bindgen/ir/context.rs

```diff
@@ -2796,6 +2797,25 @@ If you encounter an error missing from this list, please file an issue or a PR!"
             .contains(&id.into())
     }
 
+    /// Look up whether the item with `id` is type parameter after resolve.
```

Comment:
This needs a test, but it seems to me this should happen in the can_derive_copy analysis.

Reply:
@emilio Gentle reminder.